octopus-agent/src/main/java/octopus/teamcity/agent/InterruptableBuildProcess.java

```diff
@@ -15,25 +15,48 @@
 
 package octopus.teamcity.agent;
 
+import static octopus.teamcity.agent.logging.SdkLogAppender.SDK_APPENDER_NAME;
```

Comment:
:/ Exposing this is a bit nasty and it shouldn't leak from the SDK - it can be hidden, but we need to make a few tweaks (like passing in a AppenderCreator to the SDK, rather than a literal Appender).